pybop/optimisers/pints_optimisers.py

```diff
@@ -241,30 +210,33 @@ class CMAES(pints.CMAES):
 
     Parameters
     ----------
-    x0 : array_like
-        The initial parameter vector to optimize.
-    sigma0 : float, optional
-        Initial standard deviation of the sampling distribution, defaults to 0.1.
-    bounds : dict, optional
-        A dictionary with 'lower' and 'upper' keys containing arrays for lower and upper
-        bounds on the parameters. If ``None``, no bounds are enforced.
+    **optimiser_kwargs : optional
+        Valid PINTS option keys and their values, for example:
+        x0 : array_like
+            The initial parameter vector to optimize.
+        sigma0 : float
+            Initial standard deviation of the sampling distribution.
+        bounds : dict
+            A dictionary with 'lower' and 'upper' keys containing arrays for lower and
+            upper bounds on the parameters. If ``None``, no bounds are enforced.
 
     See Also
     --------
     pints.CMAES : PINTS implementation of CMA-ES algorithm.
     """
 
-    def __init__(self, x0, sigma0=0.1, bounds=None):
-        if len(x0) == 1:
+    def __init__(self, cost, **optimiser_kwargs):
+        x0 = optimiser_kwargs.pop("x0", cost.x0)
+        if x0 is not None and len(x0) == 1:
             raise ValueError(
                 "CMAES requires optimisation of >= 2 parameters at once. "
                 + "Please choose another optimiser."
             )
-        if bounds is not None:
-            self.boundaries = pints.RectangularBoundaries(
-                bounds["lower"], bounds["upper"]
-            )
-        else:
-            self.boundaries = None
+        super().__init__(cost, pints.CMAES, **optimiser_kwargs)
 
-        super().__init__(x0, sigma0, self.boundaries)
+
+class DefaultOptimiser(XNES):
```

Comment:
Instead of providing a `pybop.DefaultOptimiser` class, we can point users to the `pybop.Optimisation` API. We can discuss a bit more when I'm back, but I think that's a cleaner way to interact with PyBOP if you don't know what optimiser you want.

Reply:
Updated in #309